### PR TITLE
liveness and readiness endpoints

### PR DESCRIPTION
**Pull request by rgrotenbreg**
mostly copy/paste from the FS documentation

### CODE REVIEW

File: security/firely-auth/firely-auth-endpoints.rst [hunk 1]
```diff
@@ -117,3 +117,31 @@ Known Limitations
...
+* ``GET <base>/$readiness``
+
+These align - intentionally - with the use of liveness and readiness probes in Kubernetes.
+
+Results
+-------
+
+The ``$liveness`` operation may return one of these http status codes:
+
+#. 200 OK: Firely Server is up and running (but may still be blocked by a long running process).
```
Comment (Sieuwkje-firely) [inline security/firely-auth/firely-auth-endpoints.rst:139]: @rgrotenbreg see the comment in  https://firely.atlassian.net/browse/VONK-7337?focusedCommentId=55196
We don't really return anything other than a 200 OK response, so I am not sure if I would mention the other response types here.

File: security/firely-auth/firely-auth-endpoints.rst [hunk 2]
```diff
@@ -117,3 +117,31 @@ Known Limitations
...
+
+It can be useful to check whether Firely Auth is still up and running, and ready to handle requests. Either just for notification, or for automatic failover.
+A prominent use case is to start dependent services only after Firely Auth is up and running, e.g. in a docker-compose or in a Helm chart.
+
+Firely Auth provides two endpoints, for different purposes:
+
+* ``GET <base>/$liveness``
+* ``GET <base>/$readiness``
+
+These align - intentionally - with the use of liveness and readiness probes in Kubernetes.
```
Comment (Sieuwkje-firely) [inline security/firely-auth/firely-auth-endpoints.rst:132]: Maybe good to provide a link to the kubernetes docs here

**Review by Sieuwkje-firely — CHANGES_REQUESTED** [review]

File: security/firely-auth/firely-auth-endpoints.rst (hunk 1 above)
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-endpoints.rst]: we return 200, 402 and a component in front of FA could return a 500, like a load balancer, updated as there will be no long running process

File: security/firely-auth/firely-auth-endpoints.rst (hunk 2 above)
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-endpoints.rst]: done

**Review by Sieuwkje-firely — APPROVED** [review]

**Pull request merged**